src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSparkUnitTest.java

```diff
@@ -95,12 +96,40 @@ public void writeToLocalTest() throws IOException{
         Assert.assertTrue( FileUtils.contentEqualsIgnoreEOL(copied._2(), originalFASTQFile, StandardCharsets.UTF_8.name()) );
     }
 
+    // Test if disabling stdout stderr works
+    @Test(groups = "sv")
+    public void stdioCaptureTest() throws IOException{
```

Comment:
Do any of the existing tests actually check that the standard IO is captured when captureStdio is true? I don't see where that is.. If not please add one (you could just add assertions to an existing test, or expand this test to run again with the param set to true).